buzzmobile/tests/test_utils/listener.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python
+
+"""A ros node that will subscribe to a single message and print it to stdout.
+"""
+import cPickle as pickle
+import sys
+
+import rospy
+
+
+def listen(topic, msg_type):
+    """A function to isolate some of the subscriber logic.
+    """
+    rospy.init_node('mock_listen_'+topic.split('/')[-1], anonymous=True,
+            disable_signals=True)
+    def listener_cb(message):
+        """A default callback for our subscriber.
+        """
+        message.serialize(sys.stdout)
+        rospy.signal_shutdown('message sent')
```

Comment:
Unfortunately not, and its easy enough to do this:

    with mock_node(stuff) as mn:
        with check_topic(stuff) as ct:
            mn.send(data)
            assert about == ct.message
        with check_topic(stuff) as ct2:
            mn.send(more)
            assert about == ct2.message